packages/ui-components/style/icons/whats-new.svg

```diff
@@ -0,0 +1 @@
+<svg id="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><defs><style>.cls-1{fill:#000000;}.cls-2{fill:none;}</style></defs><title>whats-new</title><path class="cls-1" d="M26,6V8.17L5.64,11.87a2,2,0,0,0-1.64,2v4.34a2,2,0,0,0,1.64,2L8,20.56V24a2,2,0,0,0,2,2h8a2,2,0,0,0,2-2V22.74l6,1.09V26h2V6ZM18,24H10V20.93l8,1.45ZM6,18.17V13.83L26,10.2V21.8Z"/><rect id="_Transparent_Rectangle_" data-name="&lt;Transparent Rectangle&gt;" class="cls-2" width="32" height="32"/></svg>
```

Comment:
I suggest editing this svg to change the black line and match the gray in other icons we are using in the launcher (see pipeline editor and info icons)
![image](https://user-images.githubusercontent.com/25207344/183723447-e414571f-0fa1-4bd0-ad45-6841a0ad3edf.png)
Adding `class="jp-icon3 jp-icon-selectable" fill="#000000"` to the `<svg>` tag should work :)